submit/parameters.py

```diff
@@ -76,6 +78,8 @@
    fastHadd      = False                 # No fastHadd on a different T2
 nFit             = 2000                  # number of fits done in parallel
 Barrel_or_Endcap = 'ALL_PLEASE'          # Option: 'ONLY_BARREL','ONLY_ENDCAP','ALL_PLEASE'
+ContainmentCorrection = 'No' # Option: 'No', '2012reg', '2016reg','Yong'
```

Comment:
I would suggest always keeping fastHadd flag true. 

We didn't  carefully tested it when it is false, but we had problems in the code with the function in methods.py that uses normal hadd. 

Currently, although the flag is set true, we are not really using fastHadd: we modified the function in methods.py that is called when the flag is true but we made it use normal hadd (we will fix that in the future), because that method was more robust than the other where normal hadd was used. 

Marco

Inviato da iPhone

> Il giorno 22 giu 2016, alle ore 18:09, Emanuele Di Marco <notifications@github.com> ha scritto:
> 
> In submit/parameters.py:
> 
> > @@ -76,6 +78,8 @@
> >     fastHadd      = False                 # No fastHadd on a different T2
> >  nFit             = 2000                  # number of fits done in parallel
> >  Barrel_or_Endcap = 'ALL_PLEASE'          # Option: 'ONLY_BARREL','ONLY_ENDCAP','ALL_PLEASE'
> > +ContainmentCorrection = 'No' # Option: 'No', '2012reg', '2016reg','Yong'
> I would just put here '2012reg', since it is the one that we are currently using. 
> As soon as we have the new sample and the new training we switch to 2016reg
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>